enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionResult.tsx

```diff
@@ -93,7 +93,13 @@ export const InteractiveQuestionResult = ({
           <BackButton />
           {withTitle && (customTitle ?? <Title />)}
           {withResetButton && <QuestionResetButton />}
-          <FilterButton onClick={() => setQuestionView("filter")} />
+          <FilterButton
+            onClick={() =>
+              setQuestionView(
+                questionView === "filter" ? "visualization" : "filter",
```

Comment:
realized that this button should toggle between the two states, not just bring us to the filter pane